.github/workflows/release.yml

```diff
@@ -0,0 +1,36 @@
+name: "Build and push artifacts"
+on:
+  push:
+    tags:
+      - '*'
+
+jobs:
+  build:
+    name: Release artifacts
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set version
+        run: |
+          tag_name="${GITHUB_REF##*/}"
+          tag_value="${tag_name:1}-ice36-b${{ github.run_id }}"
+          echo "omero.version=$tag_value" >> etc/local.properties
+      - name: Build and package
+        run: ./build.py build-dev release-all release-src
+      - name: Upload Release Assets
+        run: |
+          set -x
+          assets=()
+          touch SHASUMS
```

Comment:
How does the MD5 file that's in your tag get created? Do we need both?

Reply:
@sbesson and I discussed it and decided to remove the MD5